### DISCUSSION TITLE
Filter out URLs in Kibana dashboard

### DISCUSSION THREAD

**Discussion by Mato816** [canarytrace/documentation#191] [Synthetic] [answered]
Hello,

i would like to ask you if there is some option to filter out only URLs from smoke test in **Smoke Overview** dashboard.
I tried to use Kibana filters to filter out just URLs containing some specific labels (smoke) but if i select combo-box in **Select smoke URL (canarytrace)** visualization there are still URLs got from user-journey and smoke tests.

Thank you.

![smoke](https://github.com/canarytrace/documentation/assets/100913907/54148313-4e39-4194-9e9f-b4816ba66744)

**Comment by rdpanek** [comment, marked as answer]
Hi @Mato816  ,
Kibana: Input controls are deprecated and will be removed in a future release. Use the new Controls to filter and interact with your dashboard data.

Please create a new control:
<img width="1800" alt="Snímek obrazovky 2023-08-25 v 10 06 26" src="https://github.com/canarytrace/documentation/assets/230124/dcc66755-1268-475e-94ea-4a8554f81a2c">
<img width="1800" alt="Snímek obrazovky 2023-08-25 v 10 05 59" src="https://github.com/canarytrace/documentation/assets/230124/b0ee4221-1120-4c05-a2a8-7b485beb7ee7">

**Comment by Mato816** [comment]
Thank you
